msal-objc-articles/reference/Classes/class_m_s_a_l_authentication_scheme_bearer.md

```diff
@@ -0,0 +1,16 @@
+---
+title: MSALAuthenticationSchemeBearer
+
+---
+
+# MSALAuthenticationSchemeBearer
+
+
+
+
+
+Inherits from <MSALAuthenticationSchemeProtocolInternal>
```

Comment:
Do the class names in the `<` and `>` need to be escaped? When I view the rendered page, I think the browser (or GitHub?) is misinterpreting this as an HTML element and nothing is displayed